apps/src/code-studio/progressReduxSelectors.js

```diff
@@ -125,6 +125,8 @@ export const getLevelPropertiesPath = state => {
       sublevelPosition = currentLevel.levelNumber;
     }
 
+    // TODO: TEACH-1864
+    // use /courses/:course_name/units/:unit_position/... instead of /s/
```

Comment:
do you have a sense of where this url will be used, and when we want to do this work (e.g. before experiment launch?) Same questions for other new Jira tasks.